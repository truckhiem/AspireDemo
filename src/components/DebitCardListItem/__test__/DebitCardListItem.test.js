import React from 'react';
import {shallow} from 'enzyme';
import {DebitCardListItem} from '../index';

const icDefault = require('res/images/ic_top_up.png');

describe('DebitCardListItem', () => {
  it('should render DebitCardListItem', () => {
    const wrapper = shallow(
      <DebitCardListItem
        icon={icDefault}
        title="DebitCardListItem"
        description="DebitCardListItem"
        onSwitch={() => {}}
        isEnable={true}
        isHaveSwitch={false}
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
