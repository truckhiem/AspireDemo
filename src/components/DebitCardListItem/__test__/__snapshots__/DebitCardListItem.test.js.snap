// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DebitCardListItem should render DebitCardListItem 1`] = `
<ForwardRef
  style={
    Object {
      "flexDirection": "row",
      "marginBottom": 32,
      "marginHorizontal": 24,
    }
  }
>
  <AspireImage
    source={null}
  />
  <View
    style={
      Object {
        "flex": 1,
        "marginLeft": 12,
      }
    }
  >
    <AspireText
      color="#25345F"
      size={14}
      text="DebitCardListItem"
    />
    <AspireText
      color="#22222266"
      size={13}
      text="DebitCardListItem"
      textStyle={
        Object {
          "height": 35,
        }
      }
    />
  </View>
</ForwardRef>
`;
